index.js

```diff
@@ -71,7 +71,11 @@ const main = async () => {
     core.setOutput("task-arn", taskArn);
 
     core.debug("Waiting for task to finish...");
-    await ecs.waitFor("tasksStopped", { cluster, tasks: [taskArn] }).promise();
+    await ecs.waitFor("tasksStopped", {
+      cluster,
+      tasks: [taskArn],
+      $waiter: { maxAttempts: 300 },
+    }).promise();
```

Comment:
From what I can see in their `.d` file, this has the signature:
```
waitFor(state: "tasksStopped", params: ECS.Types.DescribeTasksRequest & {$waiter?: WaiterConfiguration}, callback?: (err: AWSError, data: ECS.Types.DescribeTasksResponse) => void): Request<ECS.Types.DescribeTasksResponse, AWSError>;
```

So, I presume this should work. How do you think we should test this change?

Reply:
I am curious what is the default value, which is currently insufficient.

Reply:
> https://github.com/trustfractal/aws-ecs-run-task/pull/1#discussion_r1015574967

100

Reply:
How do you think we should test this?